go.mod

```diff
@@ -4,14 +4,14 @@ go 1.21
 
 require (
 	github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358
-	github.com/alecthomas/chroma v0.10.0
+	github.com/alecthomas/chroma v0.11.1
```

Comment:
There's no `v0.11.1` for Chroma, let's keep as `v0.10.0`

https://github.com/alecthomas/chroma/releases/tag/v0.10.0